src/watts/plugin_accert.py

```
@@ -0,0 +1,98 @@
+                 extra_template_inputs: Optional[List[PathLike]] = None,
+                 show_stdout: bool = False, show_stderr: bool = False):
+        try:
+            executable = Path(os.environ['ACCERT_DIR']) / 'Main.py'
+        except KeyError:
+            raise OSError("ACCERT_DIR environment variable needs to be set which contains the Main.py")
+        execute_command = [sys.executable, '{self.executable}', '-i','{self.input_name}']
+        super().__init__(executable, execute_command, template_file, extra_inputs, extra_template_inputs, show_stdout, show_stderr)
+        self.input_name = "ACCERT_input.son"
+
+
+class ResultsACCERT(Results):
+    """ACCERT simulation results
+
+    Parameters
+    ----------
+    params
+        Parameters used to generate inputs
+    name
+        Name of workflow producing results
+    time
+        Time at which workflow was run
+    inputs
+        List of input files
+    outputs
+        List of output files
+    total_cost
+        ACCERT results of total cost
+    account_table
+        ACCERT results of account table
+    """
+
+    def __init__(self, params: Parameters, name: str, time: datetime,
+                 inputs: List[Path], outputs: List[Path]):
+        super().__init__(params, name, time, inputs, outputs)
+        self.account_table = self._get_account_table()
+        self.total_cost = self.total_cost()
+
+    def total_cost(self):
+        updated_accounts= self.account_table
+        total_cost = updated_accounts['total_cost'].values[0]
+        return total_cost
+
+    def _get_account_table(self)->pd.DataFrame:
+        """Read ACCERT '.xlsx' file and return results' table
+
+        Returns
+        -------
+        Results from ACCERT_updated_account.xlsx
+
+        """
+        account_table = pd.DataFrame()
+        if Path('output.out').exists():
+            account_table = pd.read_excel('ACCERT_updated_account.xlsx')
+        return account_table
```

In the case that output.out doesn't exist, it seems like it would be better to just raise an exception rather than return the user an empty dataframe

---

Added exception